src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -145,7 +146,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
             else
             {
                 // start the regular schedule
-                StartTimer(DateTime.Now);
+                StartTimer(DateTimeOffset.Now);
```

Comment:
I wonder why we've never used the "now" local captured above here?

Reply:
Hmm, yeah, me too. Maybe because when we first captured it and now could make the interval incorrect? I'm hesitant to touch this part at this point...